tests/acceptance/async_test_base.py

```diff
@@ -92,10 +94,12 @@ def on_open_error(self, connection, error):
 
     def on_timeout(self):
         """called when stuck waiting for connection to close"""
-        self.logger.info('on_timeout called at %s', datetime.utcnow())
-        # Initiate cleanup
-        self.timeout = None  # the dispatcher should have removed it
+        self.logger.info('on_timeout at %s', datetime.utcnow())
```

Comment:
Let's have a single error log message that gives us useful information for debugging; e.g.,

```
        self.logger.error('%s timed out; on_timeout called at %s', 
            self, datetime.utcnow())
```

and get rid of `self.logger.error('on_timeout')` below